src/frontend/src/pages/DRCPage/DayCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Card, CardContent, Grid, IconButton, Typography } from '@mui/material';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import { DesignReview } from 'shared';
+
+interface DayCardProps {
+  myDate: Date;
+  events: DesignReview[] | undefined;
```

Comment:
make this, `events: DesignReview[]`, then wherever you pass in props just pass in `[]` if the object you're trying to pass in is undefined